pkg/util/testing/wrappers.go

```diff
@@ -235,6 +235,18 @@ func (j *JobWrapper) Subdomain(subdomain string) *JobWrapper {
 	return j
 }
 
+// JobSpec sets the job spec.
+func (j *JobWrapper) JobSpec(jobSpec batchv1.JobSpec) *JobWrapper {
+	j.Spec = jobSpec
+	return j
+}
+
+// JobStatus sets the job spec.
+func (j *JobWrapper) JobStatus(jobSpec batchv1.JobStatus) *JobWrapper {
+	j.Status = jobSpec
+	return j
+}
```

Comment:
instead of those, pls add wrapper functions to set specific fields in spec and status:

```
Parallelism(value int)
Completions(value int)
Ready(value int)
Succeeded(value int)
```